index.html

```diff
@@ -1,17 +1,59 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <link rel="stylesheet" href="light.css">
-    <title>First</title>
-  </head>
-  <body>
-  <h1>Моя первая страница</h1>
-  <div id="text"><a href="about/index.html">Об авторе</a>
-    <div id="light">
-      <div id="rope">
-      </div>
-    </div>
-  </div>
-  </body>
+<head>
+    <meta charset="UTF-8"/>
+    <link rel="stylesheet" href="styles/style.css">
+    <title>Fifth</title>
+</head>
+<body>
+<div>
+    <div class="Text"></div>
+    <h1>What do we offer</h1>
+    <hr>
+</div>
+<div class="main">
+    <div class="box headphones">
+        <img src="images/headphones.png" alt="HEADPHONES">
+        <p> Support 24/7</p>
+    </div>
+    <div class="box chat">
+        <img src="images/chat.png" alt="CHAT">
+        <p> Communicate directly </p>
+    </div>
+    <div class="box laptop">
+        <img src="images/laptop.png" alt="LAPTOP">
+        <p> Book online</p>
+    </div>
+    <div class="box ok">
+        <img src="images/ok.png" alt="REVIEWS">
+        <p> Real guest reviews</p>
+    </div>
+</div>
+<div class="Blue">
```

Comment:
bad neme for class, it doesn't described for what you use it. better .background_wrapper
and never use capital letter in nemes of classes

Reply:
renamed